popup.html

```diff
@@ -5,7 +5,10 @@
       </style>
     </head>
     <body>
-      <input type="checkbox" id="active"/> Activate timer
+      <label for="reload-int-input">Refresh interval (seconds)</label>
+      <input type="number" min='1' id="reload-int-input" />
```

Comment:
I still would like a slider instead of an input box. Do you think it is possible for you to change this?

Reply:
I personally think an input box gives more flexibility to the user for setting custom intervals over a larger range of numbers than sliding. What do you think is an acceptable range for the slider? I'm thinking of use cases where users would refresh over longer periods of time like 5 minutes (or 300 seconds).

Reply:
Let us have it in minutes probably! Or if you can think of some way of not having that wide input box that is good looking, that could work too!

Reply:
You don't think there would be a lot of uses under 1 minute? I could try adding some CSS to it too if you want.

Reply:
Sure that could be great too!

Reply:
What about this? I could make a slider that ranges from 1 to 60, and add a checkbox (styled as a toggle switch) that toggles either seconds or minutes? That way, it allows for a wider range of intervals while keeping the slider small-ish.

Reply:
Cool yeah, provide a slider and a unit changer beside it